roles/setup_dbt2/tasks/dbt2_install_packages_client.yml

```diff
@@ -1,17 +1,20 @@
 ---
-- name: Install packages for DBT-2 Client
-  ansible.builtin.package:
-    name: dbt2-common
-    state: present
-  when: ansible_os_family == "RedHat"
-  become: true
-
-- name: Install additional supporting packages
+- name: Install DBT-2 Client supporting packages
   ansible.builtin.package:
     name:
```

Comment:
probably should add sysstat here.  i don't know why we didn't think of that before.